ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -340,6 +340,28 @@ def test_human_output(self):
         assert returncode == 0
         assert output == "No issues found."
 
+    def test_ipa_healthcheck_fips_enabled(self):
+        """
+        Test if FIPS is enabled and the check exists.
+
+        https://pagure.io/freeipa/issue/8951
+        """
+        returncode, output = run_healthcheck(self.master, output_type="json",
+                                             failures_only=False)
+        assert returncode == 0
+
+        cmd = self.master.run_command(['fips-mode-setup', '--is-enabled'],
+                                      raiseonerr=False)
```

Comment:
If you pass source ipahealthcheck/meta/core and check MetaCheck to run_healthcheck then it will only run the check that contains the fips output so there will be less to go through and a bit faster.